CONVERTER_SERVICE/src/controller/MLController.js

```diff
@@ -0,0 +1,45 @@
+require("dotenv").config("../../.env");
+const zipPath = process.env.ZIP_PATH;
+const path = require("path");
+const Zip = require("../middleware/zipping");
+const VideoServices = require("../middleware/videoService");
+const uploadFileMiddleware = require("../middleware/uploadFilesWithoutHush");
+require("dotenv").config("../../.env");
+
+const framesZipML = async (req, res) => {
+    await uploadFileMiddleware(req, res);
+    const nameFile = req.file.filename;
+    console.log(nameFile);
+    const resultName = Date.now();
+    const videoServices = new VideoServices(req.body, nameFile, resultName);
+
+    const resultPathFrames = await videoServices.obtainFrames();
+    const zip = new Zip();
+    const pathFramesZip =
+        resultPathFrames != "" ? await zip.zipImages(resultPathFrames) : "";
```

Comment:
Maybe you can use this condition:
`resultPathFrames ? await zip.zipImages(resultPathFrames) : "";`
This value `""` it's equal to false according to JS

Reply:
Maybe so, but for now I want to show the comparison explicitly to understand the flow that the code follows.